crates/it-json-serde/src/errors.rs

```diff
@@ -14,8 +14,13 @@
  * limitations under the License.
  */
 
-mod ivalues_to_json;
-mod json_to_ivalues;
+use thiserror::Error;
 
-pub(crate) use ivalues_to_json::ivalues_to_json;
-pub(crate) use json_to_ivalues::json_to_ivalues;
+#[derive(Debug, Error)]
+pub enum ItJsonSerdeError {
```

Comment:
Suggest to rename it to `ITJsonSeDeError`, because 
- we usually write IT with both capital letters
- on other parts for serialization only short `se` is used